src/app/errorPageStyling.tsx

```diff
@@ -0,0 +1,31 @@
+import Paper from "@mui/material/Paper";
+import styled from "styled-components";
```

Comment:
"use client"?

Reply:
Shouldn't be needed this is just a styling page!